crates/bevy_ui/src/ui_node.rs

```diff
@@ -507,10 +522,16 @@ impl Default for FlexDirection {
 #[derive(Copy, Clone, PartialEq, Eq, Debug, Serialize, Deserialize, Reflect)]
 #[reflect(PartialEq, Serialize, Deserialize)]
 pub enum JustifyContent {
+    /// Items are packed toward the start of the axis
+    Start,
+    /// Items are packed toward the end of the axis
+    End,
     /// Pushed towards the start
     FlexStart,
     /// Pushed towards the end
     FlexEnd,
+    /// Items are stretched to fill the container
+    Stretch,
```

Comment:
Should document that this has no effect on flexbox containers (only affects grid)

Reply:
Deleted the stretch variant, if it's not useful for flex it shouldn't be part of this PR.